keyboards/wolf/freja65/info.json

```diff
@@ -0,0 +1,214 @@
+            { "flags": 5, "x": 165, "y": 64 },
+            { "flags": 5, "x": 114, "y": 64 },
+            { "flags": 5, "x": 104, "y": 64 },
+            { "flags": 5, "x": 44, "y": 64 },
+            { "flags": 5, "x": 23, "y": 64 },
+            { "flags": 5, "x": 3, "y": 64 },
+            { "flags": 5, "x": 10, "y": 50 },
+            { "flags": 5, "x": 0, "y": 0 },
+            { "flags": 4, "matrix": [0, 1], "x": 16, "y": 0 },
+            { "flags": 4, "matrix": [0, 2], "x": 33, "y": 8 },
+            { "flags": 4, "matrix": [0, 3], "x": 49, "y": 8 },
+            { "flags": 4, "matrix": [0, 4], "x": 65, "y": 8 },
+            { "flags": 4, "matrix": [0, 5], "x": 81, "y": 8 },
+            { "flags": 4, "matrix": [0, 6], "x": 98, "y": 8 },
+            { "flags": 4, "matrix": [0, 7], "x": 114, "y": 8 },
+            { "flags": 4, "matrix": [0, 8], "x": 130, "y": 8 },
+            { "flags": 4, "matrix": [0, 9], "x": 146, "y": 8 },
+            { "flags": 4, "matrix": [0, 10], "x": 163, "y": 8 },
+            { "flags": 4, "matrix": [0, 11], "x": 179, "y": 8 },
+            { "flags": 4, "matrix": [0, 12], "x": 195, "y": 8 },
+            { "flags": 5, "x": 220, "y": 8 },
+            { "flags": 5, "x": 244, "y": 8 },
+            { "flags": 3, "x": 223, "y": 11 },
+            { "flags": 3, "x": 234, "y": 14 },
+            { "flags": 3, "x": 234, "y": 26 },
+            { "flags": 3, "x": 234, "y": 39 },
+            { "flags": 3, "x": 228, "y": 53 },
+            { "flags": 3, "x": 208, "y": 53 },
+            { "flags": 3, "x": 192, "y": 53 },
+            { "flags": 3, "x": 175, "y": 53 },
+            { "flags": 3, "x": 158, "y": 53 },
+            { "flags": 2, "x": 140, "y": 53 },
+            { "flags": 2, "x": 123, "y": 53 },
+            { "flags": 3, "x": 106, "y": 53 },
+            { "flags": 2, "x": 88, "y": 53 },
+            { "flags": 2, "x": 76, "y": 53 },
+            { "flags": 2, "x": 54, "y": 53 },
+            { "flags": 2, "x": 37, "y": 53 },
+            { "flags": 3, "x": 17, "y": 53 },
+            { "flags": 3, "x": 4, "y": 53 },
+            { "flags": 3, "x": 4, "y": 39 },
+            { "flags": 3, "x": 4, "y": 25 },
+            { "flags": 3, "x": 4, "y": 11 },
+            { "flags": 2, "x": 21, "y": 11 },
+            { "flags": 2, "x": 38, "y": 11 },
+            { "flags": 2, "x": 56, "y": 11 },
+            { "flags": 2, "x": 73, "y": 11 },
+            { "flags": 2, "x": 90, "y": 11 },
+            { "flags": 2, "x": 107, "y": 11 },
+            { "flags": 2, "x": 124, "y": 11 },
+            { "flags": 2, "x": 141, "y": 11 },
+            { "flags": 2, "x": 158, "y": 11 },
+            { "flags": 2, "x": 176, "y": 11 },
+            { "flags": 2, "x": 190, "y": 11 },
+            { "flags": 2, "x": 206, "y": 11 }
+        ]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0067",
+        "vid": "0x5453"
+    },
```

Comment:
```suggestion
    },
    "community_layouts": ["65_ansi_split_bs"],
```

Reply:
Do I also need to provide a keymap for each community layout?

Reply:
Currently you've only supplied one layout, which coincidently matches a community layout - hence this suggestion.

If the PCB does support more than one layout, be it in community layouts or not, then please do add it to `layouts` and also supply a `LAYOUT_all` layout in `layouts` as per PR checklist guidance.  
There isn't a requirement for each layout in `layouts` to have a keymap, though a `default` keymap is a requirement.  
You can use the `LAYOUT_all` layout in the `default` and `via` keymaps as a catch all.